docs/scos/user/features/202108.0/promotions-discounts-feature-overview.md

```diff
@@ -125,93 +125,249 @@ There are two types of discount application:
+| SPICE10| 10% off spices| Percentage| 10%|  100|  No| $30 of spices|
+| MEMBER5| 5% off for members| Percentage| 5%| 5000| Yes| |
+| STORE5| 5% off storewide if you spend $50 or more|  Percentage| 5%| 9000| Yes| |
+
+**Discounts applied in priority order:**
+
+MEMBER5 (5000): $100 - $100*0.05 = $95
+
+In the presence of exclusive discounts, all non-exclusive discounts are excluded.
+Between the exclusive discounts, the discount with the higher priority is chosen.
+
+#### Scenario 4: Multiple non-exclusive, percentage discount calculation types. Executing discounts in priority order results in cart subtotal no longer satisfying conditions for one or more subsequent discounts.
+
+Cart subtotal: €100
+
+| DISCOUNT NAME|  DESCRIPTION|  DISCOUNT TYPE| DISCOUNT AMOUNT| DISCOUNT PRIORITY| EXCLUSIVENESS| NOTES|
+| --- | --- | --- | --- | --- |--- |--- |
+| 50SOCKS|  50% off Nike socks| Percentage| 50%|  100|  No| Nike socks cost €40|
+| 15PANTS|  15% off all white pants|  Percentage| 15%|100|  No| White pants cost €60|
+| SAVE20| 20% off purchases over €100|  Percentage| 20%|  300|  No| |
+| SITE5|  5% off everything in the store| Percentage| 5%| 9999| No| |
+
+**Discounts applied in priority order:**
+
+1. 50SOCKS (100 priority): €100 - €40*0.5 = €80
+2. 15PANTS (100 priority): €80 - €60*0.15 = €71
+3. SAVE20 (300 priority): No longer applies, because discounts with higher priorities have reduced the subtotal below the discount condition threshold of €100.
+4. SITE5: (9999 priority): €71 - €3.55 = €67.45
+
+#### Scenario 5: Non-exclusive, fixed amount and percentage discounts with the same priority values 
+
+Cart subtotal: €100
+
+| DISCOUNT NAME|  DESCRIPTION|  DISCOUNT TYPE| DISCOUNT AMOUNT| DISCOUNT PRIORITY| EXCLUSIVENESS|
+| --- | --- | --- | --- | --- |--- |
+| 50SOCKS|  10% off Nike socks| Percentage| 10%|  100|  No| Nike socks cost €40|
+| 15PANTS|  Save €20 on all white pants|  Fixed amount| €20|100|  No| White pants cost €60|
+
+**Discounts applied in priority order:**
+
+1. 50SOCKS (100 priority): €100 - €40*0.1 = €96
+2. 15PANTS (100 priority): €96 - €20 = €76
+
+
+**Calculation as displayed in cart:**
+
+Subtotal: €100.00
+
+50SOCKS: -€4
+15PANTS: -€20.00
+
+Grand total: €76.00
+
+#### Scenario 6: Exclusive and non-exclusive discounts without the priorities
+
+Cart subtotal: €100
+
+| DISCOUNT NAME  | DESCRIPTION| DISCOUNT TYPE | DISCOUNT AMOUNT | DISCOUNT PRIORITY|  EXCLUSIVENESS | NOTES |
+| --- | --- | --- | --- | --- |--- |--- |
+|50SOCKS | 10% off Nike socks | Percentage | 10% | |Exclusive | Nike socks cost €40 |
+|15PANTS|Save €5 on all pants| Fixed amount | €5 | | Exclusive |  |
+|SITE5|10% off everything in the store| Percentage | €10 | |  Non-exclusive | €10 | |
+
+In the presence of exclusive discounts, all non-exclusive discounts are excluded.
+Since the remaining exclusive discounts do not have priorities and the discount 15PANTS providers more free value than the discount 50SOCKS, this discount is applied.
```

Comment:
```suggestion
Since the remaining exclusive discounts do not have priorities and the discount 5PANTS provides more free value than the discount 10SOCKS, this discount is applied.
```